lib/config.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
```

Comment:
the new env vars in here all need documentation

I'd really recommend using https://github.com/logdna/env-config-node for this, it's purpose-built for this.

```js
const config = require('./config.js')
config.validateEnvVars()

if ( /*event has hostname*/ ) {
  config.set('hostname', event_host)
}

// etc
```

Reply:
I will add this after finishing testing